service/header/header.go

```diff
@@ -41,3 +42,14 @@ func (eh *ExtendedHeader) UnmarshalBinary(data []byte) error {
 	*eh = *out
 	return nil
 }
+
+// ExtendedHeaderRequest represents a request for one or more
+// ExtendedHeaders from the Celestia network.
+type ExtendedHeaderRequest struct {
+	Origin int64 // block height from which to begin retrieving headers (in descending order)
```

Comment:
Q: is there a reason to use `int64` instead of `int`? If you're going to use a non-`int`, might as well use `uint64`?

Reply:
I'm against using `int` in all cases. I think uint64 makes the most sense. I would simply keep it consistent with tendermint.

Reply:
FWIW, I'm against `int` completely and totally. `uint64` should be used almost everywhere IMO 😂.

More to the point, what does TM use? cc @renaynay

Reply:
int > uint64 in Golang for reasons. 
Consistency with tendermint? Nah...

Reply:
int64 is unfortunately consistent with Tendermint, so what's the consensus here? Do we want to be consistent and not have to cast everywhere or do we want to be correct lol.

Reply:
cc @liamsi @adlerjohn

Reply:
I'm OK with consistency here.

Reply:
Okay good to know, removed `ExtendedHeaderRequst` from this PR anyway, but good to reference in the future.